agents/ppo.py

```diff
@@ -152,30 +159,31 @@ def learn(self) -> Optional[Tensor]:
         )
         if not learn_trajectories:
             return None
-        # TODO[Zhogov Alexandr] think if norm is sane at all
-        learn_norm_rewards = get_norm_rewards(learn_trajectories)
-        for trajectory, norm_rewards in zip(learn_trajectories, learn_norm_rewards):
-            loss += self._trajectory_loss(trajectory, norm_rewards)
+        for trajectory in learn_trajectories:
+            loss += self._trajectory_loss(trajectory)
         loss /= len(learn_trajectories)
         self._optimizer.step(loss)
         return loss.detach().item()
 
-    def _trajectory_loss(self, trajectory, norm_rewards):
-        reward = np.array(norm_rewards)
+    def _trajectory_loss(self, trajectory):
```

Comment:
Please add shape descriptions/hints for this function (the same is applied for dqn.py)